all.json

```diff
@@ -1,6 +1,9 @@
 {
 	"allow": [],
 	"deny": [
+		"connectverification.com",
+		"wlfastsadaps.fastminnetafix.com",
```

Comment:
I would normally suggest the top-level as well, but Chrome is already blocking this, so probably overkill.

Reply:
yep, gonna add the top level as well later. Thanks for reminding!